src/Executable.js

```diff
@@ -105,6 +111,63 @@ export default class Executable {
         return this;
     }
 
+    /**
+     * @param {number} minBackoff
+     */
+    setMinBackoff(minBackoff) {
+        if (this._maxBackoff != null && minBackoff >= this._maxBackoff) {
+            throw new Error(
+                "minBackoff cannot be larger than or equal to maxBackoff."
+            );
+        }
+        this._minBackoff = minBackoff;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get minBackoff() {
+        return this._minBackoff;
+    }
+
+    /**
+     * @param {number} maxBackoff
+     */
+    setMaxBackoff(maxBackoff) {
+        if (this._minBackoff != null && maxBackoff <= this._minBackoff) {
```

Comment:
This should throw an error if `maxBackoff` is null. Have a default of null is fine for `maxBackoff` but if the user explicitly sets it it should be with a non-null value.